OpenSim/Simulation/Model/GeometryPath.cpp

```diff
@@ -28,9 +28,11 @@
 #include "ConditionalPathPoint.h"
 #include "MovingPathPoint.h"
 #include "PointForceDirection.h"
-#include <OpenSim/Simulation/Wrap/PathWrap.h>
 #include "Model.h"
 
+#include <OpenSim/Common/Assertion.h>
```

Comment:
Headers rearranged so that those with a matching style are clustered+alphabetical

(normalizing the header usage in OpenSim is a separate, larger, issue)